components/reddit/reddit.app.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  type: "app",
+  app: "reddit",
+  methods: {
+    _accessToken() {
+      return this.$auth.oauth_access_token;
+    },
+    _apiUrl() {
+      return `https://oauth.reddit.com`;
+    },
+    async _makeRequest(opts) {
+      if (!opts.headers) opts.headers = {};
+      opts.headers.authorization = `Bearer ${this._accessToken()}`;
+      opts.headers["user-agent"] = "@PipedreamHQ/pipedream v0.1";
+      const { path } = opts;
+      delete opts.path;
+      opts.url = `${this._apiUrl()}${path[0] === "/" ? "" : "/"}${path}`;
+      return await require("@pipedreamhq/platform").axios(this, opts);
+    },
+
+    async getNewSubredditLinks(after_link, subreddit) {
+      const newSubredditLinks = await this._makeRequest({
```

Comment:
Do you need to paginate here? What happens if 300 links have been added to a subreddit since the last time the source ran? Does Reddit have a limit on the number of items they'll return per page?

Reply:
Yes. Reddit limit is 100. I implemented pagination, can we continue this conversation in the commented related to the timestamp?